.npmrc

```diff
@@ -1,2 +1,3 @@
 package-lock=false
 auto-install-peers=true
+enable-pre-post-scripts=true
```

Comment:
[By default, pnpm doesn't run pre- and post-scripts](https://pnpm.io/cli/run#enable-pre-post-scripts), so I'm enabling this behavior.